stix_shifter_utils/stix_translation/src/utils/unmapped_attribute_stripper.py

```diff
@@ -1,13 +1,15 @@
 from stix_shifter_utils.stix_translation.src.patterns.pattern_objects import StartStopQualifier, ObservationExpression, \
-    ComparisonExpression, Pattern, CombinedComparisonExpression, CombinedObservationExpression
+    ComparisonExpression, Pattern, CombinedComparisonExpression, CombinedObservationExpression, \
+    ComparisonComparators,  ComparisonExpressionOperators, ObservationOperators
```

Comment:
Are these imports used?

Reply:
not really. I probably imported them for some testing and forgot to remove